code/game/objects/items/devices/radio/radio.dm

```diff
@@ -240,8 +240,9 @@
 	var/turf/position = get_turf(src)
 
 	var/jammed = FALSE
-	for(var/obj/item/device/jammer/jammer in GLOB.active_jammers)
-		if(get_dist(position,get_turf(jammer)) < jammer.range)
+	for(var/i in GLOB.active_jammers)
+		var/datum/component/jammer/jammer = i
+		if(get_dist(position,get_turf(jammer.parent)) < jammer.range)

```

Comment:
What does get_dist() return if one of the objects is null or in nullspace?

Reply:
no clue but it caps at 127 so nullspace should be fine.

Reply:
But if it returns 0 then a jammer in nullspace will jam every radio everywhere

Reply:
Does this check for z-level anywhere?

Reply:
that would be fairly comedical and I'm not sure how to fix that. I guess fix it by not having things go to nullspace :^)

Reply:
also get_dist is a byond proc I have no clue if it checks for z-level

Reply:
I know from experience that it does not, but does the rest of the code here?

Reply:
Oh whoops forgot to check Z here. Yeah add z check please.